lib/matplotlib/path.py

```diff
@@ -269,16 +280,14 @@ def readonly(self):
         """
         return self._readonly
 
-    def __copy__(self):
+    def copy(self):
```

Comment:
IMO this was a bug. `copy.copy` delegates to `__copy__` so it entered an infinite loop. This is unrelated to this change, but I thought it might be worth fixing.

Reply:
I would just leave things as is.  This basically means that path copying never worked, and was likely never used before (as no one noticed); we can just remove the copy() method.
After removing it copying will actually work, but will be spelt `copy.copy(path)` (like for any Python object).